src/tests/class.test.js

```diff
@@ -1 +1,79 @@
-test.skip('skip', () => {});
+/* eslint-disable no-underscore-dangle */
+import request from 'supertest';
+import app from '../app';
+import Class from '../models/class';
+import classSeed from '../seeds/class';
+
+const mockClass = {
+  day: classSeed[1].day,
+  hour: classSeed[1].hour,
+  trainer: classSeed[1].trainer,
+  activity: classSeed[1].activity,
+  slots: classSeed[1].slots,
+};
+
+const secondMockClass = {
+  day: 'Wednesdayyy',
+  hour: '17:24',
+  trainer: '646428fc0b6aa64a90624c73',
+  activity: '6465095739daaa5e7a21023a',
+  slots: 3,
+};
+
+const mockIdNotFound = '646a353ecdc4cb3be6be10b3';
+
+const invalidId = 'asdasdasd';
+
+beforeAll(async () => {
+  await Class.collection.insertMany(classSeed);
+});
+
+describe('PUT /api/classes', () => {
+  // test('should return status 400 due to existing data in DB', async () => {
+  //   const response = await request(app).put(`/api/classes/${classSeed[1]._id}`).send(mockClass);
```

Comment:
We should make this little change to make this test pass

```suggestion
  //   const response = await request(app).put(`/api/classes/${classSeed[1]._id.toString()}`).send(mockClass);
```

Reply:
I found the change it is not necessary, test will pass without adding it!